game-core/src/main/java/games/strategy/engine/framework/AutoSaveFileUtils.java

```diff
@@ -17,57 +17,47 @@
 /**
  * Provides methods for getting the names of auto-save files periodically generated during a game.
  */
-public final class AutoSaveFileUtils {
-  private AutoSaveFileUtils() {}
-
+public class AutoSaveFileUtils {
   @VisibleForTesting
-  static File getAutoSaveFile(final String fileName) {
-    return ClientSetting.saveGamesFolderPath.getValueOrThrow().resolve(Paths.get("autoSave", fileName)).toFile();
-  }
-
-  private static File getAutoSaveFile(final String baseFileName, final boolean headless) {
-    return getAutoSaveFile(getAutoSaveFileName(baseFileName, headless));
+  File getAutoSaveFile(final String baseFileName) {
+    return ClientSetting.saveGamesFolderPath.getValueOrThrow()
+        .resolve(Paths.get("autoSave", getAutoSaveFileName(baseFileName))).toFile();
   }
 
   @VisibleForTesting
-  static String getAutoSaveFileName(final String baseFileName, final boolean headless) {
-    if (headless) {
-      final String prefix = System.getProperty(TRIPLEA_NAME, "");
-      if (!prefix.isEmpty()) {
-        return prefix + "_" + baseFileName;
-      }
-    }
+  String getAutoSaveFileName(final String baseFileName) {
     return baseFileName;
   }
 
-  public static File getHeadlessAutoSaveFile() {
-    return getAutoSaveFile(addExtension("autosave"), true);
+  public File getOddRoundAutoSaveFile() {
+    return getAutoSaveFile(addExtension("autosave_round_odd"));
   }
 
-  public static File getOddRoundAutoSaveFile(final boolean headless) {
-    return getAutoSaveFile(addExtension("autosave_round_odd"), headless);
+  public File getEvenRoundAutoSaveFile() {
+    return getAutoSaveFile(addExtension("autosave_round_even"));
   }
 
-  public static File getEvenRoundAutoSaveFile(final boolean headless) {
-    return getAutoSaveFile(addExtension("autosave_round_even"), headless);
-  }
-
-  public static File getLostConnectionAutoSaveFile(final LocalDateTime localDateTime) {
+  public File getLostConnectionAutoSaveFile(final LocalDateTime localDateTime) {
     checkNotNull(localDateTime);
 
     return getAutoSaveFile(
-        addExtension("connection_lost_on_" + DateTimeFormatter.ofPattern("MMM_dd_'at'_HH_mm").format(localDateTime)));
+        addExtension("connection_lost_on_" + DateTimeFormatter.ofPattern("MMM_dd_'at'_HH_mm",
+            Locale.ENGLISH).format(localDateTime)));
```

Comment:
On my system the test cases fail because the filename doesn't contain "May" but rather "Mai" (german word for May).
Since the whole filename is in english anyways, the month name probably doesn't matter IMO.